imod/logging/logurulogger.py

```diff
@@ -0,0 +1,41 @@
+import sys
+
+from loguru import logger
+
+from imod.logging.ilogger import ILogger
+from imod.logging.loglevel import LogLevel
+
+
+class LoguruLogger(ILogger):
+    """
+    The :class:`LoguruLogger` is used to log messages using the loguru logging framework.
+    """
+
+    def __init__(
+        self,
+        log_level: LogLevel,
+        add_default_stream_handler: bool,
+        add_default_file_handler: bool,
+    ) -> None:
+        # Remove default handler set by loguru
+        logger.remove()
+
+        if add_default_stream_handler:
+            logger.add(sys.stdout, level=log_level.value)
+        if add_default_file_handler:
+            logger.add("imod-python.log", level=log_level.value)
+
+    def debug(self, message: str) -> None:
```

Comment:
you could change the type of message to "Any" and then call the logger with str(message)

Reply:
On the caller side this could be confusing. What will be logged when i provide object X? This way there can be no confusion about what is being logged